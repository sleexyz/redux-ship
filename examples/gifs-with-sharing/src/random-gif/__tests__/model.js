// @flow
import * as Test from '../../test';
import * as RandomGifModel from '../model';

Test.snapshotReduce(RandomGifModel.reduce, {
  LoadStart: {
    patch: {type: 'LoadStart'},
    state: RandomGifModel.initialState,
  },
  LoadSuccess: {
    patch: {
      type: 'LoadSuccess',
      gifUrl: 'https://media2.giphy.com/media/m7ychnf9zOVm8/giphy.gif',
    },
    state: RandomGifModel.initialState,
  },
});
