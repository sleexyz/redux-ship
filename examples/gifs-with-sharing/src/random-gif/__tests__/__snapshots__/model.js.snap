exports[`test LoadStart 1`] = `
Object {
  "gifUrl": null,
  "isLoading": true
}
`;

exports[`test LoadSuccess 1`] = `
Object {
  "gifUrl": "https://media2.giphy.com/media/m7ychnf9zOVm8/giphy.gif",
  "isLoading": false
}
`;
